### DISCUSSION TITLE
Show Originals Showing Reblogs

### DISCUSSION THREAD

**Discussion by paintingskylines** [AprilSylph/XKit-Rewritten#261] [Help Desk]
I was really excited about the new extension because I really missed having the "Show Originals" option that got defunct when Tumblr updated. Unfortunately, when I use it, I still see reblogs. It hides _some_ reblogs but it still shows some. Below is a screencap of a reblog that showed up on my dashboard while "Show Originals" was enabled. (Usernames are crossed out for privacy, as I'm not sure they would be okay with their usernames being here.) It seems to happen with text posts quite a bit, is a pattern I'm noticing. Is there any way to make sure that the extension only shows original posts or is it a problem with Tumblr? Thanks.
<img width="197" alt="screencap" src="https://user-images.githubusercontent.com/90543649/132973480-92ab003f-754a-4b32-91f4-4d9dcfc1f6fa.PNG">

**Comment by marcustyphoon** [comment]
Have you tried the checkboxes in the script's options? There is an option to show reblogs where the reblogger has added content of their own. Does disabling that option provide the behavior you're looking for?

**Reply by paintingskylines** [reply]
That seemed to fix the problem, thanks! Wasn't aware of the options

**Reply by marcustyphoon** [reply]
Awesome, no problem :)
